src/history.rs

```diff
@@ -20,7 +20,7 @@ impl History {
         History {
             entries: VecDeque::new(),
             max_len: DEFAULT_HISTORY_MAX_LEN,
-            ignore_space: true,
+            ignore_space: false,
```

Comment:
Because I checked and found no similar projet where lines which begin with a space character are ignored (`copperline`, `peterh/liner`, `haskeline`, `linenoise-ng`, `jline2`, ...).